Integrations/integration-ArcherRSA.yml

```diff
@@ -2851,6 +2913,7 @@ script:
             tz = 0;
         }
         var offset = tz ? (60 * 1000 * new Date().getTimezoneOffset()) : 0;
+        offset = tz ? parseInt(tz) : 0;
```

Comment:
Please parse the offset to a time object and check that we do the addition correctly with lastTime

Reply:
Planned on fixing the time parsing issue in a second unrelated PR, if that's ok with you